portfolio/src/main/webapp/script.js

```diff
@@ -113,3 +113,11 @@ function viewReplies() {
     }
   });
 }
+
+
+function initMap() {
+  var map = new google.maps.Map(document.getElementById('map'), {
+    zoom: 1,
+    center: {lat: 59.325, lng: 18.070}
```

Comment:
You can make these long/lat values top-level constants

Reply:
Will do!